### PR TITLE
Upgraded to 2.0 and made chapter names professional

### PR DESCRIPTION
**Pull request by bennett000**
@barretodavid I still need to address issues #10 and #11

Also I did a fresh `npm install` after an `rm -rf ./node_modules` but it wouldn't hurt to have a second attempt at running it.

This is _still_ SystemJS it was _way_ easier than upgrading to CLI@webpack.

You _must_ do an `npm start`.  `ng serve` will throw if your global version is modern.

### CODE REVIEW

File: angular-cli.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "project": {
     "version": "1.0.0-beta.10",
-    "name": "angular-training-examples"
+    "name": "angular-training-eamples"
```
Comment (barretodavid) [inline angular-cli.json]: oops `examples`

File: package.json [hunk 2]
```diff
@@ -14,34 +14,35 @@
...
-    "karma-chrome-launcher": "0.2.3",
-    "karma-jasmine": "0.3.8",
+    "codelyzer": "0.0.28",
+    "ember-cli-inject-live-reload": "1.4.1",
+    "jasmine-core": "2.5.2",
+    "jasmine-spec-reporter": "2.7.0",
+    "karma": "1.3.0",
+    "karma-chrome-launcher": "2.0.0",
+    "karma-jasmine": "1.0.2",
     "live-server": "^1.1.0",
```
Comment (barretodavid) [inline package.json]: Can we lock down this dependency as well? `"live-server": "1.1.0"`

**Review by barretodavid — COMMENTED** [review]
Looks good, a few very small changes needed

File: angular-cli.json (hunk 1 above)
Comment (bennett000) [reply angular-cli.json]: LOL

File: package.json (hunk 2 above)
Comment (bennett000) [reply package.json]: sure

**Pull request merged**